cornflow-dags/DAG/rostering/core/experiment.py

```diff
@@ -49,7 +49,13 @@ def get_objective(self) -> float:
         return self.solution.get_working_hours()
 
     def get_indicators(self) -> dict:
-        return {"fo": self.get_objective()}
+        return {
+            "objective_function": self.get_objective(),
+            "one_employee": self.get_one_employee_percentage(),
```

Comment:
The name may not be very representative, I had to look it up. Maybe better "only_one_employee" or ""pct_only_one_employee", "%_only_one_employee". At least with the % or pct in the name

Reply:
Changed